lib/spring.ts

```diff
@@ -181,3 +214,38 @@ export function configureLocalSpringBootDeploy(sdm: SoftwareDeliveryMachine) {
     );
     sdm.addCommand(ListLocalDeploys);
 }
+
+const CloudNativeReviewCommentCategories = [
+    ImportFileIoCategory,
+    ImportDotStarCategory,
+    HardcodePropertyCategory,
+    ProvidedDependencyCategory,
+];
+
+/**
+ * Review Listener to raise an issue for Cloud Native review comments.
+ */
+export const CloudNativeIssueRaisingReviewListener: ReviewListenerRegistration = {
```

Comment:
This makes the Spring pack GitHub-specific. Perhaps we need the notion of a filter in general on a ReviewListener. I guess it's fine for now